### PR TITLE
Update PR template to help auto-closing issues

### LINKED ISSUE #616: Export/Import with no passphrase set
1. Skip setting up a passphrase when setting up TextSecure
2. "Export encrpyted backup"
3. "Import encrypted backup"

TextSecure will keep asking for a passphrase. An empty passphrase is not accepted. Force-killing TextSecure and restarting it seems to resolve it.

### LINKED ISSUE #666: Change default prefereces 'Use for all SMS'
In default preferences the setting "Use for all SMS" and "Use for all MMS" should be disabled.

Think the most people would like to use SMS only as fallback ;)

### PR DESCRIPTION
**Pull request by 2-4601**
### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these browsers:
 * Ubuntu 15.10, Chrome 49.0.2623.110
 * Ubuntu 15.10, Firefox 45.0
- [x] My contribution is fully baked and ready to be merged as is
- [x] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in my commit message

----------

### Description

It's probably the eighth deadly sin to update the templates every week – or if it wasn't before, it is now. (Aaand here comes the but) but this is important as this reduces micromanaging.

Using the [`Fixes #616` syntax in PR descriptions](https://github.com/blog/1506-closing-issues-via-pull-requests) seems to only work if the PR is merged with GitHub. But as merging that way is not the preferred method at this repository, the referenced issues won't get closed automatically, and someone has to undertake the tedious task of **remembering** to close them manually.

However, the `Fixes #666` syntax also works from commit messages and without the aforementioned issue. This way also fits with the repository's established workflow. Thus we should encourage the contributor to use this method by making it a checklist item.

In short, this change will help to prevent leaving orphaned issues endlessly wandering at the repository and releases them back to the light automatically :ghost::soon:::high_brightness::sparkles:

Edit:
If I can read correctly, this new feature could also help https://github.com/blog/2141-squash-your-commits

### CODE REVIEW

**Comment by FeuRenard** [comment]
Can you please clarify where to put the `Fixes #xxxx`-part, if your PR consists of multiple commits?
- In every commit message?
- In the last commit message?
- In the first commit message?
- In any commit message?

If I observed it correctly the commits are squashed anyway. Maybe @moxie0 prefers any method to improve his workflow.

**Comment by 2-4601** [comment]
Moxie seems to pick the first commit message.

This also applies to the BitHub reward. With the current way it's enough to add `FREEBIE` just on the first commit message, though most contributors seem to add `FREEBIE` in every commit message anyway. But maybe it's too tedious to add `fixes 1234` in every commit. I guess we could clarify that the first commit message is enough.

This PR could also be obsolete if the new GitHub feature I referenced in the first post works the way I assume it works. And if it's used :)

**Comment by 2-4601** [comment]
> - [ ] If there are open issues that my contribution fixes, I am referring to them using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message of my first commit

I don't like that "If". What if there are no issues to refer to? It's doesn't feel right to tick the box.
Suggestions for better wording? :)

**Comment by FeuRenard** [comment]
I suggest the following:
> [ ] I ensure that open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/).

**Comment by moxie0** [comment]
in 3.18.0
